### DISCUSSION TITLE
Chia - Challenges from Harvesters - shows nothing on the chart

### DISCUSSION THREAD

**Discussion by myevit** [guydavis/machinaris#505] [Q&A]
Running on Truenas 0.6.7
everything seems running but there is no dots on the chart

![image](https://user-images.githubusercontent.com/13375246/146319671-eabd1f39-45d5-4a22-a7e4-510b45b81047.png)

**Comment by guydavis** [comment]
Please see basic [troubleshooting](https://github.com/guydavis/machinaris/wiki/Troubleshooting).  Check the Workers page, are your Workers configured to use the same timezone?  If not, then asking the chart to display the "last 15 minutes" of status will be off.

Then check the logs as per the troubleshooting document.  Hope this helps!

**Reply by myevit** [reply]
time zone is set correctly (date command in the container shows correct time)

In the docker log I can see only this:
```
Traceback (most recent call last):
  File "/chia-blockchain/venv/bin/chia_full_node", line 33, in <module>
    sys.exit(load_entry_point('chia-blockchain', 'console_scripts', 'chia_full_node')())
  File "/chia-blockchain/chia/server/start_full_node.py", line 60, in main
    return run_service(**kwargs)
  File "/chia-blockchain/chia/server/start_service.py", line 255, in run_service
    return asyncio.run(async_run_service(*args, **kwargs))
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 629, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.9/asyncio/base_events.py", line 596, in run_forever
    self._run_once()
  File "/usr/lib/python3.9/asyncio/base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.9/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
  File "/chia-blockchain/chia/server/start_service.py", line 193, in _accept_signal
    self.stop()
  File "/chia-blockchain/chia/server/start_service.py", line 210, in stop
    self._node._close()
  File "/chia-blockchain/chia/full_node/full_node.py", line 704, in _close
    self._transaction_queue_task.cancel()
AttributeError: 'FullNode' object has no attribute '_transaction_queue_task'

```
Chia - Partial Proofs for Pools has working graph, but Chia - Challenges from Harvesters - nothing there and timeline seems like stuck on fix line

<img width="1115" alt="Screen Shot 2021-12-16 at 2 00 57 PM" src="https://user-images.githubusercontent.com/13375246/146448407-c7bf0afc-32c0-4fd2-a00c-edbbed2a8ca8.png">

deleteing machinaris.db doesn't change anything.

If I am using TrueNas Scale machinaris chart - it's working as expected the only thing is they have outdated image in the chart, so I have to manually update the image there. It's inconvenient. Instead I am launching  it with "Launch Docker Image" with designated IP to bypass the minimum port (9000) limitation. This way it essentially creates and runs helm chart to launch docker image. But something got broken on the way.

**Reply by guydavis** [reply]
Sorry, just noticed this was still open.  Looking at the port change, I would recommend ensuring that your `controller_api_port` environment variable has been changed on all containers (fullnode, harvester, etc)  Defaults to 8927, but it looks like you indicate you changed it above.
